README.md

```diff
@@ -1,73 +1,22 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Hello-microverse
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+html and css
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: https://github.com/Danelif
+- Twitter: https://twitter.com/DanElfemuk
+- linkedIn: https://www.linkedin.com/in/dan-elifeleti-5811a3106/
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
```

Comment:
- Please, kindly update this section of your readme by doing the following:

1. Contribution header. Kindly add all the contents in the contributing header, as you've removed some contents from it. You can refer to the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to add the missing contents.
2. Show your support header.
3. Acknowledgments.
4. License.

:point_right: The arrangement should follow that used in the [microverse readme tamplate](https://github.com/microverseinc/readme-template/blob/master/README.md) to keep consistency to make your repo appear professional.